### PR TITLE
docs(gateway): cleanup references to old services

### PR DESCRIPTION
**Pull request by michaelbeaumont**

### CODE REVIEW

File: app/docs/1.8.x/explore/gateway.md [hunk 1]
```diff
@@ -364,7 +365,8 @@ These are Kuma policies so if you are running on multi-zone they need to be crea
 See the [dedicated section](/docs/{{ page.version }}/deployments/multi-zone) for detailed information.
 {% endtip %}
 
-Now define your routes which take the traffic and route it either to your `api` or your `frontend` depending on the path of the http request:
+Now let's define a `MeshGatewayRoute` to forward our traffic based on the
+matched URL path.
```
Comment (cloudjumpercat) [inline app/docs/1.8.x/explore/gateway.md:369]:
```suggestion
Now, you can define a `MeshGatewayRoute` to forward your traffic based on the
matched URL path.
```

File: app/docs/dev/explore/gateway.md [hunk 2]
```diff
@@ -224,7 +224,8 @@ Kuma gateways are configured with the [Envoy best practices for edge proxies](ht
 
 ### Usage
 
-Steps required to setup a simple gateway that exposes a http listener and 2 routes to imaginary services: "frontend" and "api".
+Let's create and configure a gateway that listens for traffic from outside of our mesh
+and forwards it to the [demo app frontend](https://kuma.io/docs/dev/quickstart/kubernetes/).
```
Comment (cloudjumpercat) [inline app/docs/dev/explore/gateway.md:228]:
```suggestion
You can create and configure a gateway that listens for traffic from outside of your mesh
and forwards it to the [demo app frontend](https://kuma.io/docs/dev/quickstart/kubernetes/).
```

File: app/docs/dev/explore/gateway.md [hunk 3]
```diff
@@ -363,7 +364,8 @@ These are Kuma policies so if you are running on multi-zone they need to be crea
 See the [dedicated section](/docs/{{ page.version }}/deployments/multi-zone) for detailed information.
 {% endtip %}
 
-Now define your routes which take the traffic and route it either to your `api` or your `frontend` depending on the path of the HTTP request:
+Now let's define a `MeshGatewayRoute` to forward our traffic based on the
+matched URL path.
```
Comment (cloudjumpercat) [inline app/docs/dev/explore/gateway.md:368]:
```suggestion
Now, you can define a `MeshGatewayRoute` to forward your traffic based on the
matched URL path.
```

**Review by cloudjumpercat — COMMENTED** [review]
Just left a few suggested copyedits!

**Review by lahabana — APPROVED** [review]

**Pull request merged**